### ISSUE TITLE
EmitDefaults with Wrapper WKTs breaks gateway.ForwardResponseMessage

### ISSUE THREAD

**Issue by Calebjh** [infobloxopen/atlas-app-toolkit#40]
Using option
```
mux := runtime.NewServeMux(append(opts, runtime.WithMarshalerOption("*", &runtime.JSONPb{OrigName: true, EmitDefaults: true}))...)
``` 
so that default values are emitted instead of omitted causes a panic in the contacts app, if a WKT is added to the .proto schema but is not set in requests.

Cause appears to have something to do with the gateway/response.go unmarshalling the response into a map, adding the 'success' response section and then remarshalling.

**Comment by Calebjh** [comment]
```2018/05/18 23:01:26 http: panic serving 172.17.0.1:60104: reflect: call of reflect.Value.Interface on zero Value
...
	/go/src/github.com/infobloxopen/atlas-contacts-app/vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/marshal_jsonpb.go:29 +0x155
github.com/infobloxopen/atlas-contacts-app/vendor/github.com/infobloxopen/atlas-app-toolkit/gateway.(*ResponseForwarder).ForwardMessage(0xc42016c3a0, 0xa5c7a0, 0xc42024faa0, 0xc4202219a0, 0xa5cea0, 0xc4201e7c80, 0xa5c160, 0xc420248620, 0xc4202fa500, 0xa59860, ...)
	/go/src/github.com/infobloxopen/atlas-contacts-app/vendor/github.com/infobloxopen/atlas-app-toolkit/gateway/response.go:113 +0x629)
...
```

points to specifically the line here https://github.com/infobloxopen/atlas-app-toolkit/blob/d7f98d00ccffd9b9cbb81bbb68f116f73b449895/gateway/response.go#L113

**Comment by Calebjh** [comment]
Fixed in release 1.4.x of grpc-gateway
